pkg/resources/fluentbit/daemonset.go

```diff
@@ -165,6 +177,19 @@ func (r *Reconciler) generateVolume() (v []corev1.Volume) {
 			},
 		},
 	}
+
+	for vCount, vMnt := range r.Logging.Spec.FluentbitSpec.ExtraVolumeMount {
+		if vMnt.Source != "" && vMnt.Destination != "" {
```

Comment:
same as above, these fields shouldn't be empty

Reply:
done